rc/extra/gdb.kak

```diff
@@ -228,12 +238,29 @@ decl -hidden str gdb_jump_client
 define-command gdb-enable-autojump %{
     set global gdb_jump_client %val{client}
 }
-
 define-command gdb-disable-autojump %{
     set global gdb_jump_client ""
 }
 
-# implementation details commands
+define-command gdb-backtrace %{
+    try %{
+        %sh{
+            if [ -n "$kak_opt_gdb_dir" ]; then
+                mkfifo "$kak_opt_gdb_dir"/backtrace
+                echo "-stack-list-frames" > "$kak_opt_gdb_dir"/pipe
+            else
+                raise
+            fi
+        }
+        edit! -fifo "%opt{gdb_dir}/backtrace" *gdb-backtrace*
+        hook -group fifo buffer BufCloseFifo .* %{
+            %sh{ rm "${kak_opt_gdb_dir}/backtrace" }
```

Comment:
May I suggest a more idiotmatic `nop %sh{ rm -f "${kak_opt_gdb_dir}/backtrace" }` here? It's not a big deal if the file isn't removed, and we don't want to take any eventual (although unlikely) output from the `rm` command.